Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM tiangolo/uwsgi-nginx-flask:python3.7
-COPY . /app
-RUN pip install -r requirements.txt
+FROM python:latest
```

Comment:
Would it be worth it to pin it to a specific version? e.g. `FROM python:3.12.3` or `FROM python:3`

Using the `latest` tag might cause issues when the underlying version changes and causes some incompatibilities, although it's quite unlikely in this case, to be fair, leave it to your choice

Reply:
HI @aalises thank you for your message. Yes that would be a good idea. Let me do that.
Btw, I just learned that you cannot use python-slim when you have a panda dependency. That's why I went with the the python base image

Reply:
@aalises I changed the`latest` tag 👍 😃 to `3.12.2-bookworm`